lib/Cron/DeleteCron.php

```diff
@@ -55,6 +58,11 @@ protected function run($argument) {
 			$this->boardMapper->delete($board);
 		}
 
+		$cards = $this->cardMapper->findToDelete();
```

Comment:
It might be useful to keep the cards to delete limited to a fixed number (e.g. 500) to keep the job execution time at a lower level and rather chunk mass deletion across multiple cron runs.

Reply:
So always just fetching the first 500 results to delete should be fine in the mapper then.

Reply:
Updated as well